pkg/s3signer/request-signature-v2.go

```diff
@@ -132,7 +127,7 @@ func PostPresignSignatureV2(policyBase64, secretAccessKey string) string {
 // CanonicalizedProtocolHeaders = <described below>
 
 // SignV2 sign the request before Do() (AWS Signature Version 2).
-func SignV2(req http.Request, accessKeyID, secretAccessKey string) *http.Request {
+func SignV2(req http.Request, accessKeyID, secretAccessKey string, vhost bool) *http.Request {
```

Comment:
rename vhost -> isVirtualHostStyle in this & preStringToSignV2() stringToSignV2() and writeCanonicalizedResource()

Reply:
> rename vhost -> isVirtualHostStyle in this & preStringToSignV2() stringToSignV2() and writeCanonicalizedResource()

Fixed.